.rubocop.yml

```diff
@@ -61,7 +62,7 @@ Style/FileName:
 Style/HashSyntax:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: hash_rockets
+  EnforcedStyle: ruby19_no_mixed_keys
```

Comment:
There's some places where the hashrocket is still required (as pointed out in that link), but the vast majority of times, you can use the new syntax.  This rule won't stop the use of the hashrocket in those circumstances (when they're needed).